app/src/main/java/ar/com/wolox/android/example/model/ItemNewsModel.kt

```diff
@@ -0,0 +1,9 @@
+package ar.com.wolox.android.example.model
+
+data class ItemNewsModel constructor(
+    val newPhoto: Int,
+    val newsTitle: String,
+    val newsContent: String,
+    val newsTime: String,
+    val newsLike: Int
+)
```

Comment:
You will need to change this names when you perform the integration with backend.